dts/bindings/qspi/st,stm32-qspi.yaml

```diff
@@ -38,11 +38,21 @@ properties:
   dmas:
     description: |
       Optional DMA channel specifier. If DMA should be used, specifier should
-      hold a phandle reference to the dma controller, the channel number,
-      the slot number, channel configuration and finally features.
+      hold a phandle reference to the dma controller not the DMAMUX even if present,
+      the channel number, the slot number, channel configuration and finally features.
+      (depending on the type of DMA: 'features' is optional)
+
+      When a DMAMUX is present and enabled, the channel is the dma one
+      (not dmamux channel) and the request is given by the DMMAUX.
 
       For example dmas for TX/RX on QSPI
          dmas = <&dma1 5 5 0x0000 0x03>;
```

Comment:
Would be nice to update this line as well, while you're at it.
So this file is fully done regarding this matter.

Reply:
description updated